src/global_logic.js

```diff
@@ -1840,6 +1840,7 @@ module.exports.getInitialTotals = function (prof, chara, summon) {
                 daBuff: 0.0,
                 taBuff: 0.0,
                 ougiGageBuff: 0.0,
+                ougiDamageChara: 0.0,
```

Comment:
Chara -> Buff here
this make missing initialize then got NaN

Reply:
Oops missed that one. Updated